CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.0.1]
+
+### Changed
+
+- Converted to monorepo and changed package name to "@metamask/snap-watch-only"
```

Comment:
Since this is not released yet you can remove this

Reply:
[e932ac9](https://github.com/MetaMask/snap-watch-only/pull/12/commits/e932ac956f923ea99b1ae9b18df00b7915c80b20)